### ISSUE TITLE
logagent模式是否还未支持？

### ISSUE THREAD

**Issue by larnsan** [white-shiro-bai/ghost_sa#4]
logagent模式是否还未支持？

**Comment by white-shiro-bai** [comment]
> logagent模式是否还未支持？

是的。不支持logagent模式。如果是不同地区部署的，可以在kafka上做文章，每个地区起自己的kafka，运行flask_main生产数据。然后在仓库所在的那个机房，起多个kafka_consumer.py去订阅每个机房的数据。这样就保证了数据的集中统一，唯一性什么的也可以得到保证。

**Comment by larnsan** [comment]
其实我想请教的问题是，使用神策服务端 SDK 将数据写入文件，然后再使用神策提供的LogAgent 工具导入到鬼策，不知道是否可行

**Comment by white-shiro-bai** [comment]
> 其实我想请教的问题是，使用神策服务端 SDK 将数据写入文件，然后再使用神策提供的LogAgent 工具导入到鬼策，不知道是否可行

 您的意思是你已经有一套官方神策在运行了，想入一套镜像在鬼策吗？可以参考这个文档，提供了完整的解决方案。我自己就是用这个迁移的我们自己的业务，很平滑，可以做到完全无感。
https://github.com/white-shiro-bai/ghost_sa/wiki/%E4%BB%8E%E7%A5%9E%E7%AD%96%E8%BF%81%E7%A7%BB%E5%88%B0%E9%AC%BC%E7%AD%96
